docs/prod-host-setup.md

```diff
@@ -1,5 +1,28 @@
 # Production Host Setup Recommendations
 
+## Jailer Configuration
+
+Using Jailer in a production Firecracker deployment is highly recommended,
+it will provide additional security boundaries for the Firecracker.
+The Jailer process applies [cgroup](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt)
+, namespace, [seccomp](https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt)
+isolation and drops privileges of the Firecracker process.
+
+Create a dedicated non-privilaged POSIX user and group to run Firecracker under.
+Use the created POSIX user and group IDs in Jailer's ``--uid <uid>`` and
+``--guid <gid>`` flags, respectively.  This will run the Firecracker as the
+created non-privileged user and group.  All file system resources used for
```

Comment:
```suggestion
created non-privileged user and group. All file system resources used for
```

Also applies to the other instances of double-space.

Reply:
Done in latest commit. Thanks